RoomDraw/app/helpers/users_helper.rb

```diff
@@ -1,2 +1,61 @@
 module UsersHelper
+  include Helper
+
+  def student_room(sid)
+    Room.find_by_sql(query(sid)).first # only one result will be returned
+  end
+
+  def groupmates(sid)
+    prettify ActiveRecord::Base.connection.execute group_query(sid)
+  end
+
+  def prettify(pg_names)
```

Comment:
Might also be nice if we offered "You're sharing a room with X" in addition to what's being provided. Not worth fixing now.